frontend/src/store/auth/reducer.ts

```diff
@@ -15,20 +15,26 @@ const initialState: State = {
 };
 
 const reducer = createReducer(initialState, (builder) => {
-  builder.addMatcher(isAnyOf(signUp.pending, signIn.pending), (state) => {
-    state.dataStatus = DataStatus.PENDING;
-  });
   builder.addMatcher(
-    isAnyOf(signUp.fulfilled, signIn.fulfilled),
+    isAnyOf(signUp.pending, signIn.pending, logout.pending),
```

Comment:
```suggestion
    isAnyOf(signUp.pending, signIn.pending),
```
you don't need to add it just because